mk/e2e.new.mk

```diff
@@ -128,10 +128,10 @@ test/e2e/debug-fast:
 # test/e2e/debug-universal is the same target as 'test/e2e/debug' but builds only 'kuma-universal' image
 # and doesn't start Kind clusters
 .PHONY: test/e2e/debug-universal
-test/e2e/debug-universal: build/kumactl images/test
+test/e2e/debug-universal: build/kumactl images/test k3d/network/create
 	$(E2E_ENV_VARS) \
 	GINKGO_EDITOR_INTEGRATION=true \
-		$(GINKGO_TEST_E2E) --keep-going=false --fail-fast $(E2E_PKG_LIST)
+		$(GINKGO_TEST_E2E) --keep-going=false --fail-fast $(UNIVERSAL_E2E_PKG_LIST)
```

Comment:
Why not keep E2E_PKG_LIST?

Reply:
It did not work for me and I just saw that `test/e2e-universal` has `UNIVERSAL_E2E_PKG_LIST ` https://github.com/kumahq/kuma/blob/006b82af7c1b1478f561bedeca0d587b85f1f005/mk/e2e.new.mk#L153

Reply:
But this is just dependant to the tests you want to run so `E2E_PKG_LIST` is generic. This target can be used with any test suite so it doesn't make sense to have it speicfic to UNIVERSAL no?

Reply:
Sorry I misread (Was looking at debug-fast :) )...